tests/packages/geckodriver.bats

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bats
+
+setup() {
+  export GECKODRIVER_VERSION="0.16.1"
+}
+
+@test "[geckodriver.sh] Installs successfully" {
+  mkdir -p "${HOME}/cache"
```

Comment:
Why do we need to create the cache directory here, and if we need, I'd add this to the `setup` function.